### ISSUE TITLE
Can we enforce dependency convergence?

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/google-cloud-eclipse#1517]
My suspicion is no, but we might be able to improve matters a little. 

http://maven.apache.org/enforcer/enforcer-rules/dependencyConvergence.html

**Comment by elharo** [comment]
Right now we have several convergence errors:

[INFO] --- maven-enforcer-plugin:1.4.1:enforce (default) @ com.google.cloud.tools.eclipse.util ---
[WARNING] 
Dependency convergence error for p2.eclipse-plugin:com.google.guava:20.0.0 paths to dependency are:
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
  +-p2.eclipse-plugin:com.google.guava:20.0.0
and
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
  +-p2.eclipse-plugin:com.google.guava:15.0.0.v201403281430

[WARNING] 
Dependency convergence error for com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT paths to dependency are:
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
and
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
  +-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0.qualifier

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Failed while enforcing releasability the error(s) are [
Dependency convergence error for p2.eclipse-plugin:com.google.guava:20.0.0 paths to dependency are:
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
  +-p2.eclipse-plugin:com.google.guava:20.0.0
and
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
  +-p2.eclipse-plugin:com.google.guava:15.0.0.v201403281430
, 
Dependency convergence error for com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT paths to dependency are:
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
and
+-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0-SNAPSHOT
  +-com.google.cloud.tools.eclipse:com.google.cloud.tools.eclipse.util:0.1.0.qualifier
]

**Comment by elharo** [comment]
I think fixing #1493 would resolve the Guava issues. Also see https://github.com/google/google-http-java-client/issues/348

**Comment by briandealwis** [comment]
I suspect the dependency convergence is confused by the Tycho target platforms (where Guava 15 is pulled from Eclipse dependencies).

**Comment by chanseokoh** [comment]
Yeah, fixing #1493 still gives the same convergence error with 20 vs 15.
